lib/rentlinx/errors.rb

```diff
@@ -7,4 +7,34 @@ class UnexpectedAttributes < RentlinxError
 
   class InvalidTypeParam < RentlinxError
   end
+
+  class BadRequest < RentlinxError
+    def initialize
+      super('The request sent to the server was invalid.')
+    end
+  end
+
+  class NotFound < RentlinxError
```

Comment:
I suggest adding another class of errors like HTTPError, that each of these (NotFound, Forbidden, and ServerError) inherit from. That adds another level of granularity for clients to rescue exceptions on.